_posts/2018-08-26-eager-dcgan/eager-dcgan.Rmd

```diff
@@ -24,26 +24,25 @@ knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
 ```
 
 
-The recent announcement of TensorFlow 2.0, much disseminated on social media, names _eager execution_ as the number one central feature of the new major version. What does this mean for R users?
+The recent announcement of TensorFlow 2.0 names _eager execution_ as the number one central feature of the new major version. What does this mean for R users?
 As demonstrated in our recent post on neural machine translation, you can use eager execution from R now already, in combination with Keras custom models and the datasets API. It's good to know you _can_ use it - but why should you? And in which cases?
 
 In this and a few upcoming posts, we want to show how eager execution can make developing models a lot easier. The degree of simplication will depend on the task - and just _how much_ easier you'll find the new way might also depend on your experience using the functional API to model more complex relationships.
-Even if you think that GANs, encoder-decoder architectures, or neural style transfer didn't pose any problems before the advent of eager execution, you might find that the alternative is a better fit to how we humans mentally picture a situation.
+Even if you think that GANs, encoder-decoder architectures, or neural style transfer didn't pose any problems before the advent of eager execution, you might find that the alternative is a better fit to how we humans mentally picture problems.
 
 For this post, we are porting code from a recent [Google Colaboratory notebook](https://colab.research.google.com/github/tensorflow/tensorflow/blob/master/tensorflow/contrib/eager/python/examples/generative_examples/dcgan.ipynb) implementing the DCGAN architecture.[@RadfordMC15]
 No prior knowledge of GANs is required - we'll keep this post practical (no maths) and focus on how to achieve your goal, mapping a simple and vivid concept into an astonishingly small number of lines of code.
 
 As in the post on machine translation with attention, we first have to cover some prerequisites.
-By the way, no need to copy out the code snippets - you'll find the complete code [here](https://github.com/rstudio/keras/tree/master/vignettes/examples/eager_dcgan.R)).
+By the way, no need to copy out the code snippets - you'll find the complete code in [eager_dcgan.R](https://github.com/rstudio/keras/tree/master/vignettes/examples/eager_dcgan.R)).
 
 
 ## Prerequisites
 
-The code in this post depends on the development versions of several of the TensorFlow R packages. You can install these packages as follows:
+The code in this post depends on the newest CRAN versions of several of the TensorFlow R packages. You can install these packages as follows:
 
 ```r
-devtools::install_github(c(
-  "rstudio/reticulate",
+install.packages(c(
```

Comment:
These need to drop the "rstudio/" preface (use just the package name)

Reply:
sorry bit tired today